App/components/Items.js

```diff
@@ -1,7 +1,7 @@
 import React from "react";
 import { StyleSheet, View, Image, TouchableOpacity, Text } from "react-native";
-
-export default function Items({ pressHandler, item }) {
```

Comment:
Why did you have to remove pressHandler here? I thought that when we are trying to press a button (in this case add items) we need to pass in pressHandler.

Reply:
My I was unaware that I deleted this functionality, it is definitely important! I will edit my file to include it!